tests/parallelizer/test_parallelizer.py

```diff
@@ -12,7 +12,7 @@
 import pytest
 import pandas as pd
 
-from dkulib.parallelizer.parallelizer import DataFrameParallelizer, ErrorHandling
+from parallelizer.parallelizer import DataFrameParallelizer, ErrorHandling
```

Comment:
Could we rename the parallelizer module to api_utils? That would better explain the context, and avoid the inelegant parallelizer.parallelizer

Reply:
Renamed👍

Reply:
Should we do the same for dku_config?

Reply:
For dku_config I'd rather not, let's keep it that way